terraform/prod.tfvars

```diff
@@ -6,8 +6,43 @@ bosh_db_multi_az = "true"
 cf_db_backup_retention_period = "35"
 cf_db_skip_final_snapshot = "false"
 cf_db_maintenance_window = "Thu:04:00-Thu:05:00"
-web_access_cidrs = "0.0.0.0/0"
-tenant_cidrs = "52.19.165.178/32,52.49.20.243/32,52.18.106.146/32,52.212.106.102/32,37.26.93.212/32,80.194.77.64/26,84.93.121.196/32,195.171.79.218/32,25.8.49.225/32,25.8.49.187/32,52.214.41.17/32,52.214.76.50/32,52.210.67.64/32"
+web_access_cidrs = [
+  "0.0.0.0/0",
+]
+tenant_cidrs = [
+  # BitZesty (trade-tarrif)
+  "52.19.165.178/32",
+  # DIT
+  "52.49.20.243/32",
+  # HMPO trial
+  "52.18.106.146/32",
+  # Usability Jenkins EIP
+  "52.212.106.102/32",
+  # Verify Jenkins server
+  "37.26.93.212/32",
+  # Shockley
+  "80.194.77.64/26",
+  # DWP carers trial
+  "84.93.121.196/32",
+  # DBS team
+  "195.171.79.218/32",
+  # MOD PaaS Trial
+  "25.8.49.225/32",
+  "25.8.49.187/32",
+  # GOV.UK Notify Jenkins server
+  "52.214.41.17/32",
+  # Accessing Government Services team Jenkins server
+  "52.214.76.50/32",
+  # GOV.UK Pay CI node
+  "52.210.67.64/32",
+  # GOV.UK Notify egress IPs (for metrics gathering)
+  "52.209.11.109",
```

Comment:
Should these new entries have a `/32` CIDR?

Not clear if it's necessarily for AWS security groups, but it would be consistent with the other entries.